### ISSUE TITLE
Initial scaffolding

### ISSUE THREAD

**Issue by JToaddler** [angular-in-action/datacenter#3]
@gnomeontherun - This repo contains the completed code base. where can i  get the initial scaffolding for data center app ? i'm trying the follow the steps in the books. it would be great if you can tell where i can find the initial scaffolding so i can follow the steps. without the initial code base its very hard to follow the steps in the book.

**Comment by gnomeontherun** [comment]
As with other chapters in the book, there is a `start` tag that let's you checkout the initial scaffolding instead of the completed version. This should be how you check it out and start using it with Git.

```
git clone https://github.com/angular-in-action/datacenter
cd datacenter
git checkout -t origin/start
```

Or you can visit the url below and download the files and skip git.

https://github.com/angular-in-action/datacenter/archive/start.zip

https://github.com/angular-in-action/datacenter/tree/start

**Comment by JToaddler** [comment]
@gnomeontherun . Thank you.
